frontend/src/lib/components/ViewRecordingButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { LemonButton, LemonButtonProps } from '@posthog/lemon-ui'
+import { useActions } from 'kea'
+import { Dayjs, dayjs } from 'lib/dayjs'
+import { IconPlayCircle } from 'lib/lemon-ui/icons'
+import { sessionPlayerModalLogic } from 'scenes/session-recordings/player/modal/sessionPlayerModalLogic'
+import { urls } from 'scenes/urls'
+
+import { EventType } from '~/types'
+
+export default function ViewRecordingButton({
+    sessionId,
+    timestamp,
+    ...props
+}: Pick<LemonButtonProps, 'size' | 'type' | 'data-attr' | 'fullWidth' | 'className'> & {
+    sessionId: string
+    timestamp?: string | Dayjs
+}): JSX.Element {
+    const { openSessionPlayer } = useActions(sessionPlayerModalLogic)
+
+    return (
+        <LemonButton
+            to={urls.replaySingle(sessionId)}
+            onClick={() => {
+                const fiveSecondsBeforeEvent = dayjs(timestamp).valueOf() - 5000
+                openSessionPlayer({ id: sessionId }, Math.max(fiveSecondsBeforeEvent, 0))
+            }}
+            sideIcon={<IconPlayCircle />}
+            {...props}
+        >
+            View recording
+        </LemonButton>
+    )
+}
+
+export const mightHaveRecording = (properties: EventType['properties']): boolean => {
+    return properties.$session_id
+        ? properties.$recording_status
+            ? properties.$recording_status === 'active'
+            : true
+        : false
```

Comment:
```suggestion
        : isSemVerAfter("1.194.0") ? false : true
```

so.... we need to consider library version here, no?

something like that - with a made up method I don't think exists 🙈

Reply:
Do we not get away without it? I was working off the assumption that `properties.$recording_status` will be undefined for older versions of the SDK. That will fail the second check in the function and hence resolve to `true` (eg. for all events in a time before we captured `$recording_status` we assume there _might_ be a recording

Reply:
ugh... yeah, ok... too many question marks and colons 🫠 

```
    return properties.$session_id
        ? properties.$recording_status
            ? properties.$recording_status === 'active'
            : true
        : false        
        ```

Reply:
too many for my brain